### PR TITLE
Rs 2023.02.22

### PR DESCRIPTION
**Pull request by rschlaikjer**
2023-02-22-194357_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 1]
```diff
@@ -146,22 +158,25 @@ In that case, I'm begging you, don't get yourself killed in a place like this.
...
-I wanted to do so many things for you. But in the end, I wasn't able to grant even a single wish.
+Even though I wanted to do so many things for her,
+ in the end,
+ I couldn't even grant that one wish.
 }
 [sha:76f73ff03352b8f3ec57ab164febb1f05f918a86]{
 -- Page 217, Offset 38276.
 -- せめて、最後に。
-So at the very least...
+That's why, as my final act―――
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:174]: >If nothing else, then as my final act―――

Maybe drop the 最後に, but it's probably fine.
>If nothing else―――

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 2]
```diff
@@ -146,22 +158,25 @@ In that case, I'm begging you, don't get yourself killed in a place like this.
...
 }
 [sha:9a4cba27fd70bbd8d1d0ef9c4a1271f1a074ecd4]{
 -- Page 217, Offset 38277.
 -- 彼女を、こんなつまらない因縁から解放するぐらい、こなさないと。
-I want to set her free from this worthless fate.
+I want to at least set her free from this worthless fate.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:179]: >I've gotta at least set her free from her wearisome fate.

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 3]
```diff
@@ -191,22 +206,24 @@ That's it.
...
 -- Page 219, Offset 38284.
 -- ……声が聞こえる。
-...I hear a voice.
+I hear a voice...
 }
 [sha:57c05c1b91df4dd995b0b58bc2b1ea6108753191]{
 -- Page 219, Offset 38285.
 -- おそらくは幻聴だろう。
-It's probably an auditory hallucination.
+I'm probably just hearing things.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:220]: Not the most graceful repetition.

1
>I hear a voice...

>But I'm probably just imagining it.

2
>Whose voice was that?

>No, I'm probably just hearing things.

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 4]
```diff
@@ -191,22 +206,24 @@ That's it.
...
 }
 [sha:5e6744a16cb8823e61c277ff8029b63cd3c6b857]{
 -- Page 220, Offset 38286.
 -- “…………遠野くん…………！”
-"......Tohno-kun......!"
+// Damn were they having a 2 for 1 on ellipses?
+"Tohno-kun!"
 }
 [sha:8ac516106feb6e3d74c374e8e7689a3ca3602dad]{
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:228]: >I hear a desperate shout...

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 5]
```diff
@@ -231,22 +251,24 @@ When I look where it's coming from, it's Senpai running toward me with her wound
...
 -- Page 221, Offset 38292.
 -- きっと、申し訳なくて、ナイフを持つ力が無くなる。
-As a result, I'm sure I'll feel guilty and lose strength in holding the knife.
+I know I'll feel too guilty to thrust my knife.
 }
 [sha:0ece556b6d6833156aeeb33c613dbbd6d88f8381]{
 -- Page 221, Offset 38293.
 -- だから、彼女がやってくる前に。
-That's why before she reaches me...
+That's why, before she reaches me―――
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:264]: ...

>So before she reaches me―――

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 6]
```diff
@@ -256,7 +278,8 @@ I put pressure into the hand holding the knife
 [sha:767aff02d7c453b588aa835c668895b73b34b9d4]{
 -- Page 222, Offset 38296.
 -- スウ、と音もなく感触さえなく、ナイフの刃が体に透る。
-Quietly, and without feeling, the blade passes through my body.
+It doesn't make a sound.
+ I don't even feel it as the blade sinks into my body.
 }
 [sha:733241b6638265f0c33c2a69888c37b15e8a6147]{
 -- Page 222, Offset 38297. Mods: @n.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:285]: These make me think that maybe 38282 should be 'That's all', but I'm not sure how much of a difference it makes. It works here better for sure, but...

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 7]
```diff
@@ -276,62 +299,62 @@ That's it.
...
 -- Page 223, Offset 38300.
 -- もう何年も前に落ちたことのある、
-I sink into the deep, deep darkness,
+Just as I did many years ago―――
 }
 [sha:9908e81ca9b0a8b032a2bc69a3e60732eb1d4050]{
 -- Page 223, Offset 38301.
 -- 深い深い闇へと落ちていけた。
-that I once fell into many years ago.
+I sink once more into the deep, deep, darkness.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:307]: Still think reversing it is preferrable
Can also move 'once' to the first clause
>I sink in the deep, deep, darkness―――

>Just as I once did many years ago.

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 8]
```diff
@@ -276,62 +299,62 @@ That's it.
...
 -- Page 224, Offset 38303.
 -- ―――声が、聞こえる。
-―――I hear a voice.
+―――I hear... a voice...
 }
 [sha:251039b21b5b273657905dcc2c37971846383b51]{
 -- Page 225, Offset 38304.
 -- “遠野くん、そんな……どうして……！”
-"Tohno-kun, why... did you..."
+"Tohno-kun, you... why!"
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:322]: question mark

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 9]
```diff
@@ -276,62 +299,62 @@ That's it.
...
 -- Page 225, Offset 38305.
 -- ―――ほら。やっぱり思ったとおり。
-―――Come on. It's just as I thought.
+―――See. I knew this would happen.
 }
 [sha:c2edc2c9c30535527724f08311bbeb72a05e8ca0]{
 -- Page 226, Offset 38306.
 -- “……やだ。こんなの、わたしはイヤです……！”
-"...No. I don't want this...!"
+"No... I don't want this!"
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01A_CIEL14A_6.txt:332]: "I don't want this" said nobody ever when they were sad.

Kind too MUCH of a stock phrase perhaps, but:
>No... Not——not like this!

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt [hunk 10]
```diff
@@ -86,57 +93,62 @@ If that were to happen―――then I would end up repeating the same mistakes a
...
 -- Page 215, Offset 38267.
 -- 我が胸の古傷にその『点』は存在する。
-There is a 'point' within the scar in my chest.
+There is a 'point' within the scar on my chest.
 }
 [sha:0db202d6598f284abe82ed3f5d6be78abda1bb46]{
 -- Page 215, Offset 38268.
 -- 俺と―――ロアの死ともいうべき『点』が。
-―――A point that spells the death of both me and Roa.
+A point that spells death for me――and for Roa.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01A_CIEL14A_6.txt:128]: Prefer what's here over both of those, a bit more dramatic.

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01A_CIEL14A_6.txt:179]: Don't love dropping the register with `gotta` in this moment

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt (hunk 10 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01A_CIEL14A_6.txt:128]: Hmmmm, sure.

File: script/Ciel/Day 14/14_01A_CIEL14A_6.txt (hunk 2 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01A_CIEL14A_6.txt:179]: Well, then you can uncontract it.
>I've got to at least

**Pull request merged**